### DISCUSSION TITLE
Teams used as groups

### DISCUSSION THREAD

**Discussion by comradekingu** [WeblateOrg/weblate#8642] [Ideas]
With the new https://hosted.weblate.org/accounts/profile/#teams functionality, it should be possible to
**call "Teams" "Groups", and allow putting more than one project in each**.

_Allows the missing functionality of navigating based on meta-categories, without having a separate system to do it._
Imagine sorting by "Android", "SailfishOS", "Health", "Crowd-sourcing", "Music", "Math", "Chess", "Communication", "Minecraft", "Nextcloud" and so on.

_It is now possible to focus on a specific platform, or get a special lense of one particular interest.
_Discovery is made easier, since it makes finding projects to translate easier by narrowing it down a lot_
There is some hope of finding a project again (among currently ~1000 on Hosted Weblate) having forgotten the name._

For the practical details, **people can apply to start and rename new groups**, confirmed by the instance admin(s)?
Maybe a role could be added, but in any event ending up with group names like "Projects that make it easier to launder money" is the thing to avoid.

Weblate also doesn't have **internal "organizations"/"translation houses"**, and it isn't something that works well elsewhere,
but the concept has merit.
Calling a group "Martins top picks" makes it easy to share with others,
advertise it, and also keep up with changes.
(Commercial translation houses organise themselves like this in various ways.
Team 1 does the translation, team 2 does the review. Different teams for different people.)

_Helps get a project off the ground by attracting a ready-made set of known translators, instead of trying to ensure someone is watching the project for every language._

Bonus 1: Option to **make a group the default view**, like https://github.com/WeblateOrg/weblate/issues/8631
This way the entire "Hosted Weblate" all of a sudden can appear to be only "Martins top picks" for whatever group of people are interested.

Bonus 2: **Automatic sorting**, by putting new Android projects in "Android", and "GPLv3" and similar in "Copyleft" for similar known info. Don't know if it makes sense to differentiate between auto-groups and custom groups.

Bonus 3: **Tying groups into the notification settings**, so members can watch this group of projects without having to click on each one, and effectively tell one group from another.

Bonus 4: **Sorting by more groups**, with negative sorting.

_If someone is interested only in AmigaOS, and "BSD-0", and not "macOS", or "CoC", then the remaining projects are dynamically sorted in the default view, available through search, without having to click "watch" to make a list._

**Comment by comradekingu** [comment]
Translation houses will have different groups for various proficiency in various fields. For example medical, and automotive level 1, 2, 3.
I am mostly interested in having a group of translators I can assign a lot of different projects to so it ends up on their watchlist, and clickable from "groups",
but it is not far away from having the good old task-based system with new projects listed that translators can be assigned to, and then reviewers,
with some progress-bars and automated e-mails here and there.

Right now it is a lot of repeated
![bilde](https://user-images.githubusercontent.com/13802408/215606160-5c5f6621-ac6b-4deb-a3d6-0bc5134c9343.png)
for me.
My thinking was more along:
![bilde](https://user-images.githubusercontent.com/13802408/215605938-ed2617fc-1f9d-443d-9533-d40c3ce6a6cc.png)

All projects with admin access in one box, expandable.
Group of translators that can be narrowed down by selecting project (UBports) or Danish within it by expanding "All languages".

**Reply by nijel** [reply]
You can have a team with access to multiple projects. But by default, each project has own teams and that definitely makes sense for individual projects as they are on Hosted Weblate.

**Reply by comradekingu** [reply]
I guess what I am after is for a project to belong to multiple teams to make it easier to distribute trust across more than one project, and to find similar projects in the now 1000+ projects on Hosted Weblate.
Currently I can have my own list of watched projects, but I can't have more than one, and I can't share it.

So, what the structure is now, plus meta-teams that can list multiple relevant projects, sometimes limited to a selected group of users.

As selected users by invitation only (people trusted by admins of https://hosted.weblate.org/teams/a-team, or to have a system of categorising the various projects. https://hosted.weblate.org/teams/sailfishos containing all SailfishOS projects for example.

Used in for example a word-cloud where things have colour and size depending on how often it is clicked and how many projects/activity it sees is more navigable than a list of 1000 projects.
